test/clj_yaml/core_test.clj

```diff
@@ -199,3 +200,23 @@ the-bin: !!binary 0101")
   (testing "emoji in comments are OK too"
     (let [yaml "# 💣 emoji in a comment\n42"]
       (is (= 42 (parse-string yaml))))))
+
+(def too-many-aliases
+  (->> (range 51)
+       (map #(str "b" % ": *a"))
+       (cons "a: &a [\"a\",\"a\"]")
+       (string/join "\n")))
+
+(deftest max-aliases-for-collections-works
+  (is (thrown? YAMLException (parse-string too-many-aliases)))
```

Comment:
Let's change this assertion to `thrown-with-msg?` and check that the exception has the string "Number of aliases" in the message.

https://bitbucket.org/asomov/snakeyaml/src/1922347d615264c50041223ec2bd689906cbcb26/src/main/java/org/yaml/snakeyaml/composer/Composer.java?at=master#Composer.java-147